### PR TITLE
Fix "game" chat tab mouseover not working from globalchat login tab.

### LINKED ISSUE #10415: Lobby / Global tabs mouseover highlight does not work when the global chat tab is activated in the lobby.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29578741-lobby-global-tabs-mouseover-highlight-does-not-work-when-the-global-chat-tab-is-activated-in-the-lobby?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### LINKED ISSUE #13277: [TS] IRC chat server messages are one word on each line.
As can be seen in this screen shot:

image

The messages from "Battlefield control" have plenty of space to be on one line but they are wrapped.

This is with TS mod and the latest build of bleed. 
OS: Windows 10
Commit ID: 618ac5838a58ed3d459975d8e540ccd7de281e42

### PR DESCRIPTION
**Pull request by pchote**
Fixes #10415.
Fixes #13277.

### CODE REVIEW

**Comment by pchote** [comment]
Rebased.

File: mods/common/chrome/lobby-globalchat.yaml [hunk 1]
```diff
@@ -88,43 +88,47 @@ Container@LOBBY_GLOBALCHAT_PANEL:
 					Height: 25
 					Text: Leave Chat
 					Font: Bold
-		Background@GLOBALCHAT_CONNECT_PANEL:
+		Container@GLOBALCHAT_MAIN_PANEL:
```
Comment (rob-v) [inline mods/common/chrome/lobby-globalchat.yaml]: Should be GLOBALCHAT_CONNECT_PANEL?

**Comment by rob-v** [comment]
It was missed in another PR, but there should be spaces in all 4 .yaml files: Width: PARENT_RIGHT-27

**Comment by pchote** [comment]
Updated.

**Review by rob-v — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]
This also fixes the crash when using `]games`.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/1a21847ac9b116d83651ec8559169201c8aa7eb5...edb6e747c716e043294239f1821b0fc9905683cb)

**Pull request merged**